### PR TITLE
Update ina219_simpletest.py

### PR DESCRIPTION
**Pull request by nvtkaszpir**
Fix minor typo.

### CODE REVIEW

File: examples/ina219_simpletest.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 i2c_bus = board.I2C()
 
-ina219 = INA219(i2c_bus)
+ina219 = INA219(i2c_bus=i2c_bus, addr=0x40)
```
Comment (tannewt) [inline examples/ina219_simpletest.py]: Why add `i2c_bus=i2c_bus`? It is a positional argument and shouldn't need the `i2c_bus=` part.

Comment (kattni) [inline examples/ina219_simpletest.py]: In addition to Scott's point above, the I2C address defaults to `0x40` in the library already, so you do not need to include the `addr=0x40` either. 

Please return this line to the following:
```suggestion
ina219 = INA219(i2c_bus)
```

**Review by kattni — CHANGES_REQUESTED** [review]

File: examples/ina219_simpletest.py (hunk 1 above)
Comment (nvtkaszpir) [reply examples/ina219_simpletest.py]: I'm used to using named parameters instead of positional ones.

**Comment by nvtkaszpir** [comment]
OK, so now it's just 'Fix minor typo' :)

**Review by kattni — APPROVED** [review]
Thank you for catching this!

**Pull request merged**